Add support for true fullscreen

kalaposfos13 opened the pull request:
SDL by default sets borderless fullscreen when calling SDL_SetWindowFullscreen, but you can manually set it to true fullscreen, if you want to. Now you can set it to true fullscreen in the config:
![image](https://github.com/user-attachments/assets/6a577bf3-bda3-407a-a5ca-ab5b38998128)
Ingame you can't switch between the modes yet, only in the launcher settings. I also didn't set up translations for it yet, as I don't have much experience with that.

kalaposfos13 committed:
Support for true fullscreen

kalaposfos13 committed:
clang

ngoquang2708 commented on src/sdl_window.cpp:
```diff
@@ -92,8 +92,22 @@ WindowSDL::WindowSDL(s32 width_, s32 height_, Input::GameController* controller_
         UNREACHABLE_MSG("Failed to create window handle: {}", SDL_GetError());
     }
 
-    SDL_SetWindowMinimumSize(window, 640, 360);
```
Why is this need to be removed?

kalaposfos13 replied:
It didn't have to be, it was a mistake

kalaposfos13 committed:
Re-add mistakenly deleted line

DanielSvoboda committed:
Size

I adjusted the size of the entire screen.
trophies font size and added a margin so it wouldn't be so spaced out.

kalaposfos13 committed:
Merge pull request #6 from DanielSvoboda/patch-1

Size

kalaposfos13 committed:
Merge remote-tracking branch 'origin/main' into fullscreen
